zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -176,27 +211,24 @@ class ZiplineLoader internal constructor(
         serializersModule,
         initializer,
       )
+
+      // If we successfully loaded local code, don't collect the remote code. We're done!
       if (localManifest != null) {
         previousManifest = localManifest
         return@channelFlow
       }
+
       manifestUrlFlow.collect { manifestUrl ->
-        if (previousManifest != null &&
-          freshnessChecker.isFresh(previousManifest!!, previousManifest!!.freshAtEpochMs ?: Long.MIN_VALUE)
```

Comment:
I’m changing it to no longer check freshness here . . . on the first load this will be the embedded code, which we already decided was fresh, so I think this might be a no-op anyway

Reply:
the idea is that the cashed code can be stale, and if that's the case, we need a reload. 

`freshnessChecker` is the hook to pass in the logic of "if the code is stale or not", `freshnessChecker` will take `shelf_life` and `minimal_commit_timestamp` into consideration